payments_service/auth/views.py

```diff
@@ -2,41 +2,47 @@
 
 from django.core.exceptions import ObjectDoesNotExist
 
-from rest_framework.decorators import api_view
 from rest_framework.response import Response
 from slumber.exceptions import HttpClientError
 
 from .. import solitude
-from ..base.views import error_400
+from ..base.views import error_400, UnprotectedAPIView
 from .forms import SignInForm
 
 log = logging.getLogger(__name__)
 
 
-@api_view(['POST'])
-def sign_in(request):
-    form = SignInForm(request.DATA)
+class SignInView(UnprotectedAPIView):
 
-    if not form.is_valid():
-        return error_400(response=form.errors)
+    def post(self, request):
+        form = SignInForm(request.DATA)
 
-    fxa_uuid = u'fxa:{u}'.format(u=form.cleaned_data['access_token']['user'])
+        if not form.is_valid():
+            return error_400(response=form.errors)
 
-    api = solitude.api()
-    status = 200
-    try:
+        fxa_uuid = (u'fxa:{u}'
+                    .format(u=form.cleaned_data['access_token']['user']))
```

Comment:
`u'fxa:{a.user}'.format(t=form.cleaned_data['access_token'])` is actually about one char shorter :)